### ISSUE TITLE
[BUG] Settings - Auto Start

### ISSUE THREAD

**Issue by zxcv10336** [eythaann/Seelen-UI#208]
Hello, can you fix the problem of automatic startup first so that the program can be used normally without affecting the experience?

**Comment by eythaann** [comment]
@zxcv10336 could you try lastest version? I add some type of workaround but if the app doesn't start after some couple seconds, send me your log files inmediatly (don't open seelen app before sending the logs after the failed start).

**Comment by eythaann** [comment]
Closing this in favor of:  #367
